### PR TITLE
Rename badly named util/filecache constants

### PR DESCRIPTION
**Pull request by pshipton**
The original names could conflict with other defines.

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile win jdk8

**Comment by pshipton** [comment]
jenkins test sanity,extended plinux jdk8

File: runtime/util/filecache.c [hunk 1]
```diff
@@ -65,11 +65,11 @@ assert_cache_separation(struct J9PortLibrary *portLibrary, J9CachedFileHandle* h
...
 
 	/* go through all the caches to get their ranges */
-	for (i = 0; i < CACHE_BUFFER_NUM; i++) {
+	for (i = 0; i < J9FILECACHE_BUFFER_NUM; i++) {
 		base = handle->cacheTable[i].cacheBase;
-		for (j = i+1; j < CACHE_BUFFER_NUM; j++) {
+		for (j = i+1; j < J9FILECACHE_BUFFER_NUM; j++) {
 			compare_to_base = handle->cacheTable[j].cacheBase;
-			if ( (base < compare_to_base) && ((base + CACHE_BUFFER_SIZE -1) >= compare_to_base) ) {
+			if ( (base < compare_to_base) && ((base + J9FILECACHE_BUFFER_SIZE -1) >= compare_to_base) ) {
```
Comment (keithc-ca) [inline runtime/util/filecache.c]: While you're here, can you add the missing spaces that should be both sides of the binary operator `-`?

File: runtime/util/filecache.h [hunk 2]
```diff
@@ -26,16 +26,12 @@
...
-#undef NO_DATA
-#endif
-#define NO_DATA -1
-#define HAS_DATA(index) handle->cacheTable[index].hwm >= 0
+#define J9FILECACHE_BUFFER_SIZE 16*1024
+#define J9FILECACHE_BUFFER_NUM 4
+#define J9FILECACHE_UNUSED FALSE
+#define J9FILECACHE_INUSE TRUE
+#define J9FILECACHE_NO_DATA -1
+#define J9FILECACHE_HAS_DATA(index) handle->cacheTable[index].hwm >= 0
```
Comment (keithc-ca) [inline runtime/util/filecache.h]: Please add the missing parentheses for `J9FILECACHE_BUFFER_SIZE`, `J9FILECACHE_NO_DATA` and `J9FILECACHE_HAS_DATA`.

Comment (pshipton) [reply runtime/util/filecache.h]: Done

File: runtime/util/filecache.c (hunk 1 above)
Comment (pshipton) [reply runtime/util/filecache.c]: Done here and other places as well.

**Comment by keithc-ca** [comment]
Jenkins test sanity,extended win32 jdk8

**Comment by keithc-ca** [comment]
Jenkins test sanity,extended plinux jdk11

**Pull request merged**